Sources/Flamingo/Utilities/Env.swift

```diff
@@ -0,0 +1,56 @@
+#if os(Linux)
+    import Glibc
+#else
+    import Foundation
+#endif
+
+/**
+  Utility to interact with environment variables.
+*/
+public struct Env {
+
+  /**
+    Returns a value for the given key.
+
+    - Parameter key: The key.
+
+    - Returns: Optional value if env var is found.
+  */
+  public static func value(_ key: String) -> String? {
+    guard let value = getenv(key), string = String(validatingUTF8: value) else {
+      return nil
+    }
+
+    return string
+  }
+
+  /**
+    Sets a new value for a given environment variable.
+
+    - Parameter key: The key.
+    - Parameter value: The value.
+  */
+  public static func add(value: String, key: String) {
```

Comment:
@RamonGilabert nothing. I see that like `ENV["key"] = "value"` looks more familiar